portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,36 @@ function addRandomGreeting() {
   // Picks a random quote and adds it to the page
 }
 
-function getData() {
-  fetch('/data').then(response => response.text()).then((msg) => {
-    document.getElementById('quote-container').innerText = msg;
+function getValues() {
+  fetch('/data').then(response => response.json()).then((msg) => {
+      var messages = String(msg);
+      messages = messages.split(",");
```

Comment:
Super super nit: Use consistent quotes throughout the file (e.g. user single quotes or double quotes throughout.